release-notes/1.0/known-issues-1.0.0.md

```diff
@@ -0,0 +1,232 @@
+// Catch the exception and call Exception.ToString() on it, e.g.
+
+try
+{
+    throw new Exception();
+}
+catch (Exception ex)
+{
+    Console.WriteLine(ex.ToString());
+}
+```
+
+<https://github.com/dotnet/coreclr/issues/5828>
+
+## Unexpected OutOfMemoryException on unix
+
+App on unix can throw OutOfMemoryException when there is still enough memory available in case the number of memory mappings it has made exceeds the `/proc/sys/vm/max_map_count`.
+
+Set the `/proc/sys/vm/max_map_count` to a larger value for applications that experience that problem or are expected to create a large number of mappings. One source of such mappings is excessive usage of `System.Reflection.Emit.AssemblyBuilder.DefineDynamicAssembly` with the access parameter set to `AssemblyBuilderAccess.Run`.
+
+## Lazily binding an assembly from Resolving event for static load can result in a crash
+
+If an app has a static assembly reference to an assembly that is missing (which is not the normal case since .NET Core app have their dependencies complete) OR attempts to do Assembly.Load against an assembly that is missing AND has wired up to the Resolving event to load the assemblies from custom locations, then the event will not return a valid reference and could result in a crash.
+
+The workaround to address the above problem is one of the following:
+
+Load the assembly using `LoadFromAssemblyName` instead of Assembly.Load, OR
+Create a custom `AssemblyLoadContext` and override its Load method to resolve the assembly.
+
+<https://github.com/dotnet/coreclr/issues/5837#issuecomment-226657996>
+
+## Nano Server TP5
+
+When working on TP5 of Nano server, users will encounter the following error if they try to run either portable or self-contained application:
+
+```
+Failed to load the dll from [C:\hwapp_s\bin\Debug\netcoreapp1.0\win10-x64\hostpolicy.dll], HRESULT: 0x8007007E
+An error occurred while loading required library hostpolicy.dll from [C:\hwapp_s\bin\Debug\netcoreapp1.0\win10-x64]
+```
+
+If you’re using Nano Server Technical Preview 5 with .NET Core CLI, due to a bug in the product, you will need to copy binaries from `c:\windows\system32\forwarders`. The destination depends on the type of deployment that you are choosing for your application.
+
+For portable applications, the forwarders need to be copied to the shared runtime location. The shared runtime can be found wherever you installed .NET Core 1.0.0 (by default, it is `C:\Program Files\dotnet`) under the following path: `shared\Microsoft.NETCore.App\<version>\`.
+
+For self-contained applications, the forwarders need to be copied over into the application folder, that is, whereever you put the published output.
+
+This process will ensure that that the dotnet host finds the appropriate APIs it needs. If your Nano Server Technical Preview 5 build is updated or serviced, please make sure to repeat this process, in case any of the DLLs have been updated as well.
+
+## dotnet does not support UTF-16 BOM
```

Comment:
We might want to make the title more clear that we mean the utility, not the framework. Replace "dotnet" with "global.json", perhaps?  (Unless it also applies to project.json?)